packages/@aws-cdk/aws-rds/README.md

```diff
@@ -673,6 +676,105 @@ new rds.ServerlessClusterFromSnapshot(this, 'Cluster', {
 });
 ```
 
+### Amazon Aurora Serverless v2
+
+Amazon Aurora Serverless v2 scales instantly to hundreds of thousands of transactions in a fraction of a second. 
+As it scales, it adjusts capacity in fine-grained increments to provide the right amount of database resources 
+that the application needs. There is no database capacity for you to manage. You pay only for the capacity your 
+application consumes, and you can save up to 90% of your database cost compared to the cost of provisioning capacity for peak load.
+
+Read [Using Aurora Serverless v2](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless-v2.html) and 
+[Getting started with Aurora Serverless v2](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless-v2.upgrade.html) to learn more about it.
+
+To create an Aurora cluster with Aurora Serverless v2 support, specify `serverlessV2Scaling` in the
+`DatabaseCluster` construct. The following example initializes an Aurora cluster with Serverless v2 support.
+
+```ts
+declare const vpc: ec2.Vpc;
+
+const cluster = new rds.DatabaseCluster(stack, 'cluster', {
+  engine: rds.DatabaseClusterEngine.auroraMysql({
+    version: rds.AuroraMysqlEngineVersion.VER_3_02_1,
+  }),
+  serverlessV2Scaling: {
+    maxCapacity: 1,
+    minCapacity: 0.5,
+  },
+  // do not create any provisioned instance in the cluster
+  instances: 0,
```

Comment:
In order to create this cluster with a serverless v2 writer we have to set `instances=0`?

Reply:
Yes 😄